fips_assessment.yml

```diff
@@ -11,6 +11,9 @@
 - name: openshift serverless fips assessment
   ansible.builtin.include_tasks: /tmp/fips-assessments/openshift-serverless/tasks/main.yml
 
+- name: openshift-pipelines fips assessment
```

Comment:
We want these to be dynamically looked up and executed based on the current operator being run, correct? I didn't think we wanted these hard-coded like this. I understand we put the first one in as an example, but I thought we were aiming for something not hard-coded.